python/lsst/qa/explorer/matchVisits.py

```diff
@@ -0,0 +1,122 @@
+"""
+"""
+import os
+import pandas as pd
+import numpy as np
+
+from lsst.pex.config import Config, Field
+from lsst.pipe.base import CmdLineTask, ArgumentParser
+from lsst.daf.persistence import NoResults
+from lsst.qa.explorer.match import match_lists
+from lsst.pipe.drivers.utils import TractDataIdContainer
+
+from .parquetTable import ParquetTable
+
+# Question: is there a way that LSST packages store data files?
+ROOT = os.path.abspath(os.path.dirname(__file__))
+
+
+class MatchVisitsConfig(Config):
+    coaddName = Field(dtype=str, default="deep", doc="Name of coadd")
+    matchRadius = Field(dtype=float, default=0.2, doc="match radius in arcseconds")
+
+
+class MatchVisitsTask(CmdLineTask):
+    """Write a tract-level table of closest-match visit IDs
+
+    Run this task on a tract, and it writes a full-tract `visitMatchTable` (a ParquetTable)
+    of coadd -> visit match ids and match distances, organized with a multi-level index.
+    """
+    _DefaultName = "matchVisits"
+    ConfigClass = MatchVisitsConfig
+
+    inputDataset = 'analysisCoaddTable_forced'
+    outputDataset = 'visitMatchTable'
+
+    @classmethod
+    def _makeArgumentParser(cls):
+        parser = ArgumentParser(name=cls._DefaultName)
+
+        parser.add_id_argument("--id", cls.inputDataset,
+                               help="data ID, e.g. --id tract=12345",
+                               ContainerClass=TractDataIdContainer)
+        return parser
+
+    def matchCats(self, df1, df2, raColumn='coord_ra', decColumn='coord_dec'):
+        """Match two catalogs, represented as dataframes
```

Comment:
Make sure your documentation format follows the current LSST, numpydoc standards: https://developer.lsst.io/python/numpydoc.html